examples/darwin-framework-tool/BUILD.gn

```diff
@@ -23,11 +23,17 @@ assert(chip_build_tools)
 config("config") {
   include_dirs = [
     ".",
+    "${chip_root}/src/darwin/Framework/CHIP/zap-generated",
+    "${chip_root}/src/darwin/Framework/CHIP",
+    "${chip_root}/examples/darwin-framework-tool/commands/common",
     "${chip_root}/zzz_generated/darwin-framework-tool",
     "${chip_root}/zzz_generated/controller-clusters",
     "${chip_root}/examples/chip-tool/commands/clusters/ComplexArgument.h",
+    "/tmp/macos_framework_output",
```

Comment:
Is this intended to have `/tmp/` output?

Reply:
It is, it's a hold over for a build location in CI, since mixing Xcode build outputs, with inputs to GN is... interesting :)

Reply:
FYI: Going to try to clean this up, once everything is cleanly building.